plugins/cassandra/cassandraembed.cpp

```diff
@@ -2783,6 +2945,13 @@ void getFieldNames(const CassandraXmlMapping *mappings, StringBuffer &names, Str
     tableName.append(mappings->columnType);
 }
 
+const char *queryTableName(const CassandraXmlMapping *mappings)
+{
+    while (mappings->columnName)
+        mappings++;
+    return mappings->columnType;
```

Comment:
trivial: It might be more consistent to use a NULL columnType to mark the tableName